File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/cimrpatsigns/d/Patparam.java
package iih.ci.mr.cimrpatsigns.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 参数DTO DTO数据 
 * 
 */
public class Patparam extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 就诊类型
	 * @return String
	 */
	public String getCode_entp() {
		return ((String) getAttrVal("Code_entp"));
	}
	/**
	 * 就诊类型
	 * @param Code_entp
	 */
	public void setCode_entp(String Code_entp) {
		setAttrVal("Code_entp", Code_entp);
	}
	/**
	 * 就诊状态
	 * @return String
	 */
	public String getSd_status() {
		return ((String) getAttrVal("Sd_status"));
	}
	/**
	 * 就诊状态
	 * @param Sd_status
	 */
	public void setSd_status(String Sd_status) {
		setAttrVal("Sd_status", Sd_status);
	}
	/**
	 * 当前科室
	 * @return String
	 */
	public String getId_dep_nur() {
		return ((String) getAttrVal("Id_dep_nur"));
	}
	/**
	 * 当前科室
	 * @param Id_dep_nur
	 */
	public void setId_dep_nur(String Id_dep_nur) {
		setAttrVal("Id_dep_nur", Id_dep_nur);
	}
	/**
	 * 过滤类型
	 * @return String
	 */
	public String getWheretype() {
		return ((String) getAttrVal("Wheretype"));
	}
	/**
	 * 过滤类型
	 * @param Wheretype
	 */
	public void setWheretype(String Wheretype) {
		setAttrVal("Wheretype", Wheretype);
	}
	/**
	 * 过滤值
	 * @return String
	 */
	public String getValue() {
		return ((String) getAttrVal("Value"));
	}
	/**
	 * 过滤值
	 * @param Value
	 */
	public void setValue(String Value) {
		setAttrVal("Value", Value);
	}
	/**
	 * 测量日期
	 * @return FDate
	 */
	public FDate getMeasuredate() {
		return ((FDate) getAttrVal("Measuredate"));
	}
	/**
	 * 测量日期
	 * @param Measuredate
	 */
	public void setMeasuredate(FDate Measuredate) {
		setAttrVal("Measuredate", Measuredate);
	}
	/**
	 * 模板ID
	 * @return String
	 */
	public String getId_mrtplvt() {
		return ((String) getAttrVal("Id_mrtplvt"));
	}
	/**
	 * 模板ID
	 * @param Id_mrtplvt
	 */
	public void setId_mrtplvt(String Id_mrtplvt) {
		setAttrVal("Id_mrtplvt", Id_mrtplvt);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.mr/cimrvt/d/Patparam.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.mr.cimrpatsigns.d
{
    /// <summary>
    /// Patparam 的摘要说明。
    /// </summary>
    public class Patparam : BaseDTO {

        public Patparam() {
 
        }

        /// <summary>
        /// 就诊类型
        /// </summary>
		public string Code_entp {
            get { return getAttrVal<string>("Code_entp",null); }
            set { setAttrVal<string>("Code_entp", value); }
        }

        /// <summary>
        /// 就诊状态
        /// </summary>
		public string Sd_status {
            get { return getAttrVal<string>("Sd_status",null); }
            set { setAttrVal<string>("Sd_status", value); }
        }

        /// <summary>
        /// 当前科室
        /// </summary>
		public string Id_dep_nur {
            get { return getAttrVal<string>("Id_dep_nur",null); }
            set { setAttrVal<string>("Id_dep_nur", value); }
        }

        /// <summary>
        /// 过滤类型
        /// </summary>
		public string Wheretype {
            get { return getAttrVal<string>("Wheretype",null); }
            set { setAttrVal<string>("Wheretype", value); }
        }

        /// <summary>
        /// 过滤值
        /// </summary>
		public string Value {
            get { return getAttrVal<string>("Value",null); }
            set { setAttrVal<string>("Value", value); }
        }

        /// <summary>
        /// 测量日期
        /// </summary>
		public DateTime? Measuredate {
            get { return getAttrVal<FDate>("Measuredate",null); }
            set { setAttrVal<FDate>("Measuredate", value); }
        }

        /// <summary>
        /// 模板ID
        /// </summary>
		public string Id_mrtplvt {
            get { return getAttrVal<string>("Id_mrtplvt",null); }
            set { setAttrVal<string>("Id_mrtplvt", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Code_entp", "Sd_status", "Id_dep_nur", "Wheretype", "Value", "Measuredate", "Id_mrtplvt"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mr.cimrpatsigns.d.Patparam";
        }
    }
}
